trace/src/main/org/jetbrains/lincheck/trace/SerializationStream.kt

```diff
@@ -130,7 +130,22 @@ private class FileStreamingThread(
     private var bufferSize = 1024
     private var buffer: ByteBuffer = ByteBuffer.allocate(INDEX_CELL_SIZE * bufferSize)
 
-    private val queue = ArrayBlockingQueue<Job>(1024)
+    private val queue = object : ArrayBlockingQueue<Job>(1024) {
```

Comment:
Please leave a comment in the source code on why this is necessary + make it a non-anonymous class for clarity.